_content/articles/century-of-confusion_green-phillip-scott-ellis.md

```diff
@@ -0,0 +1,18 @@
+---
+title: "A Century of Confusion: The Brick Reliefs of Cambodia's Phnom Trap Towers"
+authors:
+  - "Green, Phillip Scott Ellis"
+external_url: "https://buddhism.lib.ntu.edu.tw/en/search/search_detail.jsp?seq=607765&comefrom=authorinfo"
```

Comment:
Use the publisher's url not a third party. https://poj.peeters-leuven.be/content.php?url=article&id=3269004&journal_code=JIABS&download=yes